docs/update-locking.md

```diff
@@ -18,7 +18,29 @@ In supervisors >= v4.0.0 and any OS that is not Resin OS 1.X, the old lock locat
 
 ### Taking the lock
 
-Using the above-mentioned library, the lock can be acquired like in this CoffeeScript example:
+There are many different tools and libraries to provide proper lockfile functionality. Here are a few examples, and you can search online for others in other programming languages not mentioned here explicitly.
+
+#### Shell
+
+One simple way to take the lock is using [lockfile](https://linux.die.net/man/1/lockfile) (available for example in Debian from the `procmail` package) or [flock](https://linux.die.net/man/1/flock) (available for example in Debian from the `linux-utils` package). Some example use are:
```

Comment:
From my experiments, simply using `touch /tmp/resin/resin-updates/lock` worked without any additional configuration.

Reply:
That only works *sometimes*, and we want to discourage it. When using `touch`, the supervisor may have created the lock previously, in which case the app will be killed and the user will think the lock was taken.